### ISSUE TITLE
Support the latest version of RabbitMQ in h

### ISSUE THREAD

**Issue by seanh** [hypothesis/h#7642]
h currently uses RabbitMQ 3.6 which was end-of-life'd by the RabbitMQ project [in May 2018](https://www.rabbitmq.com/versions.html).

This ticket is to make sure that h works with the latest version of RabbitMQ, fix any issues necessary to get it working, and provide a PR to upgrade the development environment (though we may not choose to merge that PR yet). Actually upgrading our production RabbitMQ will then be handled separately.

Note that h shares a production RabbitMQ instance with LMS and Checkmate so support needs to be added to those apps as well before we can upgrade it. See https://github.com/hypothesis/lms/issues/4549 and https://github.com/hypothesis/checkmate/issues/577

**Comment by nairiboo** [comment]
Closing in favour of https://github.com/hypothesis/product-backlog/issues/1462
